--- a/mingw-w64-headers/crt/locale.h	2020-09-16 22:20:15.000000000 -0400
+++ b/mingw-w64-headers/crt/locale.h	2021-03-21 15:51:13.675050815 -0400
@@ -61,6 +61,16 @@
     char n_sep_by_space;
     char p_sign_posn;
     char n_sign_posn;
+#if __MSVCRT_VERSION__ >= 0xA00 || _WIN32_WINNT >= 0x601
+    wchar_t* _W_decimal_point;
+    wchar_t* _W_thousands_sep;
+    wchar_t* _W_int_curr_symbol;
+    wchar_t* _W_currency_symbol;
+    wchar_t* _W_mon_decimal_point;
+    wchar_t* _W_mon_thousands_sep;
+    wchar_t* _W_positive_sign;
+    wchar_t* _W_negative_sign;
+#endif
   };
 #endif
 
